src/api/db/data/20210520160000_set_default_for_distro_remote.rb

```diff
@@ -0,0 +1,14 @@
+class SetDefaultForDistroRemote < ActiveRecord::Migration[6.0]
+  def up
```

Comment:
@hennevogel why is this necessary? you set the default in the db migration through change_column_default already and the column is new

Reply:
https://github.com/ankane/strong_migrations#adding-a-column-with-a-default-value